R/functions.R

```diff
@@ -232,57 +231,121 @@ create_select_gene <- function(id, list, selected) {
               selectize = FALSE)
 }
 
-create_slider_cutoff <- function(id, title, start, stop, var_id){
-  if (is.null(var_id)) return()
-  if (var_id == "") {
-    sliderInput(id, title,
-                min = 1,
-                max = 1,
-                step = 0.025,
-                value = 1,
-                width = 200)
-  } else {
-    sliderInput(id, title,
-                min = 0,
-                max = 1,
-                step = 0.025,
-                value = c(start, stop),
-                width = 200)
-  }
-}
+# CREATE MANUAL COLOR SCHEME FOR A LIST ========================================
+#' @source Modified based on https://gist.github.com/peterk87/6011397
+install_packages("RColorBrewer")
```

Comment:
Do you think putting the installation of packages inside the code like here is good? Maybe not something that needs to be addressed in this PR, but: I really would like to see a proper packaging of this whole thing with a `DESCRIPTION` file. 

I started looking into this after I added support for running R packages in `mybinder.org` but didn't have time to finish it. 😞

Reply:
(I'll not comment on the other `installs` below) 😄

Reply:
I thought this will be the task for next step: packaging & docking 😉

Reply:
however, I've moved all the installation commands into global.R